extension/js/keyboardShortcuts.js

```diff
@@ -35,6 +42,10 @@ function checkKey(e) {
 		previewPost();
 	}
 
+	if (e.code == "KeyN" && e.altKey && post) {
```

Comment:
You probably want to remove this ' && post' from here since the notification functionality is not dependent on a page that has a 'post' element.

Reply:
oops will fix

Reply:
nice catch